ui/panels-plugin/src/plugins/line-chart/tooltip/tooltip-model.ts

```diff
@@ -45,7 +46,7 @@ export const defaultCursorData = {
 
 export const emptyTooltipData = {
   cursor: defaultCursorData,
-  focusedSeries: [{ seriesIdx: null, datumIdx: null, date: '', seriesName: '', x: 0, y: 0, markerColor: '' }],
+  focusedSeries: null, //[{ seriesIdx: null, datumIdx: null, date: '', seriesName: '', x: 0, y: 0, markerColor: '' }],
```

Comment:
@sjcobb to prevent the tooltip from rendering I updated `focusedSeries` to be null by default. Before I delete the old default I wanted to get your take 

It solves this little black square showing up cc @Nexucis  ![image](https://user-images.githubusercontent.com/170681/152442732-5f4dce81-d076-451d-85b9-e2216729368b.png)

Reply:
Looks good! Not sure why I did it that way in the first place

Reply:
perfect !